src/Translator/src/DependencyInjection/UxTranslatorExtension.php

```diff
@@ -43,11 +43,12 @@ public function prepend(ContainerBuilder $container)
         if (!$this->isAssetMapperAvailable($container)) {
             return;
         }
-        $config = $container->getExtensionConfig('ux_translator')[0];
+        $config = $this->processConfiguration(new Configuration(), $container->getExtensionConfig('ux_translator'));
         $container->prependExtensionConfig('framework', [
             'asset_mapper' => [
                 'paths' => [
                     __DIR__.'/../../assets/dist' => '@symfony/ux-translator',
+                    $config['dump_directory'] => 'var/translations', // Keeping for BC, see https://github.com/symfony/ux/pull/1904#issuecomment-2162066941
                     $config['dump_directory'] => '@app/translations',
```

Comment:
You're right, when we tested it worked because we had it the other way around.  
There's no way to fix this because even with 
```php
asset_mapper' => [
                'paths' => [
                    __DIR__.'/../../assets/dist' => '@symfony/ux-translator',
                    $config['dump_directory'] => '@app/translations',
                    '%kernel.project_dir%/var/translations' => 'var/translations',
                ],
            ],
```
the keys will be the same when running the default conf.